bigquery/query_places.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: default
+title: Places
+nav_order: 1
+parent: DC to BQ Sample Queries
+grand_parent: BigQuery
+---
+
+# Query Category: Places
+
+## Place containment
```

Comment:
Should we add some preamble here about replacing the name of the dataset in these queries? Similar to what we have in the generated queries (which we wanted to point here, so we should include more complete information here).

Reply:
I added the suggested sentence (personally, I am not sure if this was needed?). @beets I am not sure I follow your suggestion.